### DISCUSSION TITLE
Sparkle fails to suggest update when it is available

### DISCUSSION THREAD

**Discussion by odrobnik** [sparkle-project/Sparkle#2439] [Q&A]
Sparkle logs this on the console of a Developer ID signed app, with DSA key in the app cast.

```
Error: Serving updates without an EdDSA key and only using Apple Code Signing is deprecated and may be unsupported in a future release. Visit Sparkle's documentation for more information: https://sparkle-project.org/documentation/#3-segue-for-security-concerns
```

Question: does this mean it won't automatically check for updates? It presents the update when I manually check for updates though.

**Comment by zorgiepoo** [comment]
That is a warning when you are not using EdDSA; this does not presently prevent update checking from working. If you relied on the older DSA support it would be an error that requires you to transition to EdDSA. If checking for updates manually works, then probably the automatic checks also work.

**Reply by odrobnik** [reply]
But for some reason my users report that the dialog to show that an update is available never pops up. How can I debug that? And when I launch the app it also doesn‘t check.

**Reply by zorgiepoo** [reply]
Read https://sparkle-project.org/documentation/#6-test-sparkle-out -- if you're still having trouble, paste contents of `defaults read your-bundle-id` to examine the Sparkle defaults.
